lib/app/features/chat/messages/views/components/message_item_wrapper/message_item_wrapper.dart

```diff
@@ -13,18 +12,21 @@ class MessageItemWrapper extends HookWidget {
     required this.isMe,
     required this.child,
     required this.contentPadding,
+    this.isLastMessageFromSender = true,
     super.key,
   });
+
   final bool isMe;
   final Widget child;
+  final bool isLastMessageFromSender;
```

Comment:
Who is `sender`? It's not that obvious, consider renaming it.

Reply:
Yes, you're right, renamed to the `isLastMessageFromAuthor`. I think it's more obvious, cos we author in each message.